components/form/builder/src/Icons/IconClose.js

```diff
@@ -0,0 +1,35 @@
+import React, {memo, useEffect, useState} from 'react'
+
+var SVGComponent = function SVGComponent(_ref) {
```

Comment:
This file among other icons are unnecesarily already compiled for the src.

Reply:
It is because I don't want to use an Iconset.

Reply:
You don't need an iconset, you only have to avoid putting compiled files in the `src` folder and create a componente with the `svg` you want to use:

It would be something like this:
```
export default memo(() => (<svg viewBox='0 0 64 64'><path d='M60 62a2 2 0 0 1-1.41-.59L32 34.83 5.41 61.41a2 2 0 0 1-2.83-2.83L29.17 32 2.59 5.41a2 2 0 0 1 2.82-2.82L32 29.17 58.59 2.59a2 2 0 0 1 2.83 2.83L34.83 32l26.58 26.59A2 2 0 0 1 60 62z'></path></svg>)
))